packages/theme/package.json

```diff
@@ -50,7 +50,7 @@
   "oclif": {
     "commands": "dist/cli/commands",
     "hooks": {
-      "init": "@shopify/cli-kit/node/hooks/init"
+      "init": "@shopify/theme/node_modules/@shopify/cli-kit/dist/node/hooks/init.js"
```

Comment:
This assumes a certain structure and might break if you install the CLI globally through a package manager. For example Yarn might end up organizing the packages like:

```
@shopify/
  theme
  cli-main
  theme
```

Reply:
Yeah, I didn't like this solution, but I couldn't find a better one...

Reply:
What about moving this to the `cli-main` package? That way we ensure there's only one call.

Reply:
You mean moving the hooks from cli-kit, right? That could work, let me try...

Reply:
The implementation of the hooks remain in the `cli-kit` module, but we remove [this](https://github.com/Shopify/cli/blob/main/packages/theme/package.json#L53) across all the packages and place it in the `package.json` of `cli-main`.

Reply:
I noticed we have some inconsistencies already, for example [cli-hydrogen](https://github.com/Shopify/cli/blob/main/packages/cli-hydrogen/package.json) doesn't have the hook define. So I think `cli-main` is a more suitable place for it.

Reply:
Ah, that won't work... The hook from cli-main it's already being called (as you can see in [this log](https://user-images.githubusercontent.com/14979109/187916269-64ddb089-2323-41be-9e26-0064ff07fc76.png)). The problem is that the cli-kit is duplicated across both packages (theme and cli), so it's initializing the store in the wrong package. I can show you this in a pair session if you want.

And the hooks should stay in `cli-kit` because they are also used from `create-app` or `create-hydrogen`, which don't depend on `cli-main`.